mogp_emulator/GaussianProcessGPU.py

```diff
@@ -282,6 +370,35 @@ def D(self):
         """
         return self._densegp_gpu.D()
 
+
+    @property
+    def n_corr(self):
+        """
+        Returns number of correlation length parameters
+
+        :returns: Number of parameters
+        :rtype: int
+        """
+        return self._densegp_gpu.n_corr()
+
+    @property
+    def n_data(self):
```

Comment:
Hmm... so this is a tricky one in terms of maintaining consistency with the CPU version, since for the GPU we don't (yet) have the analytic mean...   So should `n_params` still continue to include meanfunction parameters (and we could keep `n_data` as the non-meanfunc-hyperparameters)?   
Or should we have `n_params` give the same value as for CPU implementation, and do `n_params + theta.n_mean` when we need to consider all hyperparameters?

Reply:
I would keep `n_params` as the only option and include the mean function parameters and change it if you get an analytic mean function implemented, but am not strongly bound to that opinion.

Reply:
Yep, I agree, `n_params` is used everywhere now, and includes meanfunction parameters.